src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationOptions.cs

```diff
@@ -77,7 +69,7 @@ public string Caption
         /// <value>
         /// The back channel timeout.
         /// </value>
-        public TimeSpan BackchannelTimeout { get; set; }
+        public TimeSpan BackchannelTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
So basically if you want to make them virtual, we can't use auto properties, and should set the fields to the defaults in the ctor instead of the property setter